src/bipartite.jl

```diff
@@ -0,0 +1,96 @@
+"""
+    BipartiteView{T<:Real} <: AbstractGraph{Int64}
+
```

Comment:
unnecessary newline

Reply:
changed. maybe let's edit such things directly in the code.